observations/marine/sst_generic.yaml.j2

```diff
@@ -44,3 +45,20 @@
     where:
     - variable: {name: GeoVaLs/distance_from_coast}
       minvalue: 50e3
+
+  obs post filters:
+  - filter: Background Check
+    absolute threshold: 5.0
+  - filter: Perform Action  # Palmico Sound
+    action:
+      name: accept
+    where:
+    - variable:
+        name: MetaData/latitude
+      minvalue: 34.5
+      maxvalue: 39.7
```

Comment:
What's the reason for the larger maxvalue for latitude compared to the bounding box?

Reply:
I'll update the comment, the box includes the `P. sound` and the `C. bay`. I already forgot how to spell the `P. Sound` and I could never spell the `C. Bay` ...